src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,18 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck ford = new Truck();
+          ford.doWork();
+          ford.stopWork();
+        Bulldozer graider = new Bulldozer();
+          graider.doWork();
+          graider.stopWork();
+        Excavator CAT = new Excavator();
```

Comment:
```suggestion
        Excavator cat = new Excavator();
```
з капсом називаються константи (static final змінні), для простих змінних потрібно використовував camelCase неймінг